dev-docs/acceptance-testing/acceptance_checklist.md

```diff
@@ -0,0 +1,86 @@
+# acceptance checklist
```

Comment:
i guess this should say superteam acceptance checklist...same for file name. or domain services acceptance checklist